api/room/models.py

```diff
@@ -24,12 +33,14 @@ class Room(Base, Utility):
     wing_id = Column(Integer, ForeignKey('wings.id'))
     cancellation_duration = Column(Integer, default=10)
     floor = relationship('Floor')
-    resources = relationship(
-        'Resource', cascade="all, delete-orphan",
-        order_by="func.lower(Resource.name)")
+    resources = relationship('Resource', cascade="all, delete-orphan")
```

Comment:
Hello @bevkololi , I don't know if you have rebased with develop already. If you have, please remember to add back the `order_by` here. Otherwise nice work.